src/main/java/edu/byu/cs/autograder/Grader.java

```diff
@@ -65,16 +66,11 @@ public Grader(String repoUrl, String netId, GradingObserver observer, Phase phas
         File stageRepo = new File(stagePath, "repo");
 
         // Init Grading Context
-        int requiredCommits = 10;
-        int requiredDaysWithCommits = 3;
-        int commitVerificationPenaltyPct = 10;
-        int minimumChangedLinesPerCommit = 5;
-
+        var cvConfig = PhaseUtils.verificationConfig(phase);
```

Comment:
Do you think this would be easier to read without using `var`?

Reply:
Sure, that's fine.

I am used to relying on the inferred typing information more frequently, but it's OK to have it explicitly defined.

Reply:
After looking into it more, I see why I didn't include the type explicitly here. `PhaseUtils.verificationConfig()` returns a specific type, and then this type is directly passed through into `GradingContext`. If the type were to change, I wouldn't need to change this line at all, though I would change the place it's generated and the place it's used.

Since there's no explicit need to have the variable, typing it as `var` makes it more versatile to changes.

As a matter of face, I think I'll just in-line the reference and avoid making a variable in the first place.

Reply:
Except, that having the explicit type information emphasizes the ability of future developers to get that same type from some source other than PhaseUtils. It doesn't have to come from there, and they may combine config values from different places or different fields in order to construct the actual result.

I'll include the type information.